avm-transpiler/src/transpile.rs

```diff
@@ -70,7 +70,11 @@ pub fn brillig_to_avm(brillig: &Brillig) -> Vec<u8> {
                 lhs,
                 rhs,
             } => {
-                assert!(is_integral_bit_size(*bit_size), "BinaryIntOp bit size should be integral: {:?}", brillig_instr);
+                assert!(
```

Comment:
I think we weren't running cargo fmt on this file :)

Reply:
ouch, I was just working on it but I guess if I run cargo ftm as well I'll be ok  :D